src/uu/cp/src/cp.rs

```diff
@@ -279,6 +279,21 @@ static PRESERVABLE_ATTRIBUTES: &[&str] = &[
     "all",
 ];
 
+static CP_UPDATE_LONG_HELP: &str =
```

Comment:
we added support for long help in the md files
See https://github.com/uutils/coreutils/blob/main/src/uu/stat/stat.md?plain=1#L9